MapleServer2/AIScripts/AiHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MapleServer2.Types;
+
+namespace MapleServer2.AIScripts;
+
+public static class AiHelper
+{
+    public static readonly Dictionary<string, AIState> AIStates = new()
+    {
+        {
+            "AI_DefaultNew.xml",
+            new DefaultNew()
+        },
+        {
+            "AI_PrisonGovernor2_battle.xml",
+            new PrisonGovernor2Battle()
+        }
+    };
```

Comment:
hmm is there a way to automate this? Could we just parse the ai script name from the NPC xmls without file extension and name the AI script exactly as it is?
Just thinking ahead when there's dozens and dozens more.